github/util.py

```diff
@@ -41,12 +41,14 @@ def __init__(
         github: GitHub,
```

Comment:
this hsould better be named github_api (to be more expressive and consitent with other places in our code base; or: replace with github_cfg